ledger/common/bitutils/utils.go

```diff
@@ -12,8 +12,18 @@ func Bit(b []byte, idx int) int {
 // SetBit sets the bit at position i in the byte array b
```

Comment:
```suggestion
// SetBit sets the bit at position i in the byte array b to 0 if the value is 0, otherwise, sets it to 1
```

Reply:
Following [Ramtin's and Tarak's suggestion](https://github.com/onflow/flow-go/pull/1800#discussion_r778338959), I reverted `SetBit` back to its previous definition https://github.com/onflow/flow-go/blob/3426d41a814a6f658e23363e6281e0702fa02a9a/ledger/common/bitutils/utils.go#L24-L26
and added the corresponding method for setting a bit to `0`
https://github.com/onflow/flow-go/blob/3426d41a814a6f658e23363e6281e0702fa02a9a/ledger/common/bitutils/utils.go#L33-L35


The generalized method, where the user specifies the bit value is now called `WriteBit` to avoid confusion:  https://github.com/onflow/flow-go/blob/3426d41a814a6f658e23363e6281e0702fa02a9a/ledger/common/bitutils/utils.go#L11-L16